unstructured/partition/pdf_image/pdfminer_processing.py

```diff
@@ -134,3 +137,55 @@ def merge_inferred_with_extracted_layout(
         inferred_page.elements[:] = elements
 
     return inferred_document_layout
+
+
+@requires_dependencies("unstructured_inference")
+def clean_pdfminer_inner_elements(document: "DocumentLayout") -> "DocumentLayout":
+    """Clean pdfminer elements from inside tables and stores them in extra_info dictionary
+    with the table id as key"""
```

Comment:
could this be a lint thing rather than enforced-by-human? i think this pattern exists all over the place in the repo but sure, this one could be updated. [imo, not a high prioirity thing]

Reply:
Doesn't work well as a lint thing unfortunately. Docstrings are prose so such a linter that was really effective would need a language model and no one has come up with that yet as far as I know.

As far as priority, I just change ones I happen to be working with and always use PEP 257 in new code. I don't try to do any wholesale updating. That satisfies me. Boy scout rule basically (always leave the code cleaner than you found it) :)